pkg/kudoctl/bundle/finder/bundle_finder_test.go

```diff
@@ -0,0 +1,46 @@
+package finder
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestManager_GetBundle(t *testing.T) {
+	f := &Manager{
+		local: NewLocal(),
+		uri:   nil,
+	}
+	b, err := f.GetBundle("../testdata/zk", "")
```

Comment:
why is the manager abbreviated to `f`? 🤔

Reply:
because it is a finder...  I can switch it out